### DISCUSSION TITLE
Is it possible to downgrade?

### DISCUSSION THREAD

**Discussion by n-ce** [vite-pwa/vite-plugin-pwa#729] [Q&A] [answered]
I want to downgrade from 0.20.0 to 0.19.8

Page loading on old ios devices is broken 
I'm suspecting the new update is behind it as this wasn't the issue before.
![photo1720105028](https://github.com/vite-pwa/vite-plugin-pwa/assets/69345507/56f4f70a-2184-48f5-8eba-e395ef69c9e0)

Reasons of Suspicion
- #726 
- I am personally on Android, where this isn't an issue but the first update i pushed to the 0.20.0 SW put the deployment into the same blank state for which I had to reload and then it was fine.

So, is it possible to downgrade safely?

**Comment by userquin** [comment, marked as answer]
You can fix the version, use "0.19.8" in your package.json and install deps.
